lib/node_modules/@stdlib/math/base/special/asec/benchmark/c/native/benchmark.c

```diff
@@ -90,16 +90,19 @@ static double rand_double( void ) {
 * @return elapsed time in seconds
 */
 static double benchmark( void ) {
+	double x[ 100 ];
 	double elapsed;
-	double x;
 	double y;
 	double t;
 	int i;
 
+	for ( i = 0; i < 100; i++ ) {
+		x[ i ] = ( 100.0*rand_double() ) - 50.0;
```

Comment:
```suggestion
		x[ i ] = ( 1000.0*rand_double() ) + 1.0;
```

Domain was incorrect.